tests/test_eload_validation.py

```diff
@@ -78,7 +78,14 @@ def test_parse_vcf_check_report(self):
         assert nb_warning == 1
 
     def test_structural_variant(self):
-        self.assertListEqual([True, False, True, True], self.sv_validation._detect_structural_variant())
+        assert self.sv_validation.eload_cfg.query('validation', 'structural_variant_check', 'files', 'test1.vcf',
```

Comment:
You need to still call `detect_structural_variant` here...

If you want to test that the method does the correct detection and also sets the config correctly, I would recommend completely removing the [structural variant section](https://github.com/EBIvariation/eva-submission/blob/005db9a2cfed7793767f8beead49c8670e42338a/tests/resources/eloads/ELOAD_70/.ELOAD_70_config.yml#L17) from the test eload config, so you can ensure it's starting from a clean state.  You can use the setup/teardown methods to restore the config after each test run (like [this](https://github.com/EBIvariation/eva-submission/blob/005db9a2cfed7793767f8beead49c8670e42338a/tests/test_eload_validation.py#L24)).

Reply:
I am sorry..I missed this earlier...I have updated the code now...I have removed the contents related to SV from the config file and then ran the test and tested whether the config file is getting populated corrected or not.